haskell-customize.el

```diff
@@ -324,6 +324,17 @@ The default is `haskell-interactive-prompt' with the last > replaced with |."
   :type 'string
   :group 'haskell-interactive)
 
+(defcustom haskell-interactive-use-interactive-prompt t
```

Comment:
I really dislike forcing users to know about some magical defcustoms. Is there a way to get the same functionality to Just Work without any fuss like setting something to some value?